sdk/js/src/Client.ts

```diff
@@ -75,6 +75,15 @@ export class DVCClient implements Client {
                 this.eventEmitter.emitError(err)
                 return this
             })
+            
+        if (!options?.reactNative && typeof window !== 'undefined') {
+            window.addEventListener('message', (event) => {
```

Comment:
Can you also unregister this listener in the `close` method?

Reply:
sorry, I'll make a PR to add this!